src/main/java/com/ray/message/dao/MessageDao.java

```diff
@@ -0,0 +1,48 @@
+package com.ray.message.dao;
+
+import com.ray.message.model.Message;
+import org.apache.ibatis.annotations.Insert;
+import org.apache.ibatis.annotations.Mapper;
+import org.apache.ibatis.annotations.Select;
+import org.apache.ibatis.annotations.Update;
+import org.springframework.stereotype.Repository;
+
+import java.util.Date;
+import java.util.List;
+
+@Mapper
+@Repository
+public interface MessageDao {
+
```

Comment:
nitpick: consider capitalizing all keywords in SQL. Like `INSERT INTO `message` (...) VALUES (...)`